mlflow/__init__.py

```diff
@@ -191,6 +193,7 @@
     "get_artifact_uri",
     "get_tracking_uri",
     "set_tracking_uri",
+    "is_tracking_uri_set",
```

Comment:
This method was previously only exposed in `mlflow.tracking`; now that we're removing the API docs, I'm adding it to fluent so that it's documented in https://mlflow.org/docs/latest/python_api/mlflow.html.